vizro-core/tests/unit/vizro/models/_navigation/test_accordion.py

```diff
@@ -13,41 +14,39 @@
 class TestAccordionInstantiation:
     """Tests accordion model instantiation."""
 
-    def test_create_accordion_default(self):
-        accordion = Accordion(id="accordion_id")
-        assert accordion.id == "accordion_id"
-        assert accordion.pages == ["Page 1", "Page 2"]
-
-    def test_create_accordion_pages_as_list(self, pages_as_list):
+    def test_accordion_valid_pages_as_list(self, pages_as_list):
         accordion = Accordion(pages=pages_as_list, id="accordion_id")
         assert accordion.id == "accordion_id"
-        assert accordion.pages == pages_as_list
+        assert accordion.pages == {"SELECT PAGE": pages_as_list}
 
-    def test_create_accordion_pages_as_dict(self, pages_as_dict):
+    def test_accordion_valid_pages_as_dict(self, pages_as_dict):
         accordion = Accordion(pages=pages_as_dict, id="accordion_id")
         assert accordion.id == "accordion_id"
         assert accordion.pages == pages_as_dict
 
-    def test_field_invalid_pages_input_type(self):
-        with pytest.raises(ValidationError, match="2 validation errors for Accordion"):
-            Accordion(pages=[vm.Button()])
+    def test_navigation_valid_pages_not_all_included(self):
```

Comment:
What does "not all included" mean?

Reply:
If you don't include all registered pages into the Navigation, e.g. let's say you have a code as below:

`Dashboard(pages=[Page(title="Page 1", ...), Page(title="Page 2", ...)]], navigation=Navigation(pages=["Page 1"]))`

You have two registered pages, but only include one in the navigation. What will be visible in the navigation is the one page you included into the navigation

Reply:
Ok, I was a fan of the warning, but if its too complicated to maintain, not too fussed about it 👍